tools/clang/test/DXC/help_text.test

```diff
@@ -0,0 +1,7 @@
+// Test /? and /help option
+// RUN: %dxc /? | FileCheck %s
+// RUN: %dxc /help | FileCheck %s
```

Comment:
```suggestion
// Test -? and -help option
// RUN: %dxc -? | FileCheck %s
// RUN: %dxc -help | FileCheck %s
```

Seems dxc generally prefers to use `-` rather than `/` for flags.

Reply:
Do you suggest that I change the / usage here?  It works?  Should I also be testing - ?